docs/platform/release-notes/tizen-5-0-m1.md

```diff
@@ -317,8 +317,8 @@ Release date: May. 31, 2018
     - “capi-media-camera-tool” must be installed to use “camera_test”.
 - Recorder
   - A related feature is changed.
-    - As-Is: <http://tizen.org/feature/camera>, <http://tizen.org/feature/microphone>
-    - To-Be: <http://tizen.org/feature/media.audio_recording>, <http://tizen.org/feature/media.video_recording>
+    - As-Is: \<http://tizen.org/feature/camera\>, \<http://tizen.org/feature/microphone\>

```

Comment:
Suggestion. How about applying \` to these features to prevent the link?
For now the \\ for \> are regarded as actual typo, not escape character, 
so they are causing link error.

![image](https://user-images.githubusercontent.com/9958374/64604189-af6ebc00-d3fc-11e9-8858-fc11f64c529b.png)

![image](https://user-images.githubusercontent.com/9958374/64604437-1be9bb00-d3fd-11e9-9aae-ba8d75b8e90e.png)

Reply:
They are GitHub rendering errors.
Please see https://docs.stg.tizen.org/staging/824/platform/release-notes/tizen-5-0-m1.

Reply:
\<tizen... \> was changed to `<tizen...>`.